packages/frontend/src/components/common/Filters/FilterInputs/FilterStringAutoComplete.tsx

```diff
@@ -161,7 +161,7 @@ const FilterStringAutoComplete: FC<Props> = ({
                     <Tooltip
                         withinPortal
                         position="left"
-                        label={`Click here to cache results now`}
+                        label={`Click here to refresh cache filter values`}
```

Comment:
quick small copy change suggested by Jake. More UI changes will be added on a separate PR